root/static/scripts/guess-case/MB/GuessCase/Main.js

```diff
@@ -123,6 +122,10 @@ const flags = require('../../flags');
     // because additional behavior isn't needed.
     MB.GuessCase.series = MB.GuessCase.work;
 
+    // Event doesn't have it's own handler, and just uses the work handler
+    // because additional behavior isn't needed.
+    MB.GuessCase.event = MB.GuessCase.work;
```

Comment:
… this below the `MB.GuessCase.series = MB.GuessCase.work;` line.

But again, @bitmap or @gentlecat might know better. :)